guides/common/modules/proc_upgrading-el7-to-el8.adoc

```diff
@@ -164,6 +232,18 @@ If this happens, do the following to clean up packages that cannot automatically
 ----
 # journalctl -u leapp_resume -f
 ----
+endif::[]
+
+ifdef::satellite[]
+. Reboot the system.
+
+. After the system reboots, a live system conducts the upgrade, reboots to fix SELinux labels, then reboots into the final {RHEL} 8 system.
```

Comment:
This is more of a description what's happening and no "step" that a user has to take. Either reword to "Wait until ..." or use this as description for the first point.